smart-contracts/contracts/interfaces/ERC721.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.4.24;
 * @title ERC721 interface (http://erc721.org/)
 */
 
-contract ERC721 {
+interface ERC721 {
```

Comment:
It would probably make sense to check that this is the actual most recent interface used for ERC721... I believe the one I used is not 100% identical to what OpenZeppelin does.

Reply:
The `interface` vs `contract` change does align with the latest standard on http://erc721.org/ but OpenZeppelin uses a `contract` in order to compose from multiple interfaces.  I can roll this back, and then per above we then explore using OpenZeppelin directly instead of a copy paste.  If we need to diverge from their standard, we may be able to still use some of the interfaces/contracts from OpenZeppelin used to create their ERC721.